lib/utils/getDeviceInfo.js

```diff
@@ -25,7 +25,13 @@ async function getDevicesDetails({authContext}, devicesIds) {
 		devices = [...devices, ...response.values];
 	}
 
-	const usedDevices = filter(d => devicesIds.includes(d.deviceId), devices);
+	let usedDevices = [];
+
+	devicesList.forEach(x => {
```

Comment:
I think that using `reduce` function is more semanticly correct in this scope of code.